test/workers/service_discovery/create_service_worker_test.rb

```diff
@@ -5,7 +5,6 @@
 module ServiceDiscovery
   class CreateServiceWorkerTest < ActiveSupport::TestCase
     setup do
-      ThreeScale.config.service_discovery.stubs(enabled: true)
```

Comment:
This was caught when `stubbing_method_unnecessarily` config was enabled for Mocha.